.swiftlint.yml

```diff
@@ -0,0 +1,86 @@
+# Updated for v0.27.0
+
+disabled_rules:
+  # We run the swift_combined_lint_run_script.sh which handles TODO/FIXME/WARN/WARNING independently, so we want to disable this.
+  - todo # TODOs and FIXMEs should be resolved.
+
+  
+opt_in_rules:
+  # - missing_docs # Declarations should be documented.
+  - anyobject_protocol # Prefer using AnyObject over class for class-only protocols.
+  - attributes # Attributes should be on their own lines in functions and types, but on the same line as variables and imports.
+  - closure_end_indentation # Closure end should have the same indentation as the line that started it.
+  - closure_spacing # Closure expressions should have a single space inside each brace.
+  - conditional_returns_on_newline # Conditional statements should always return on the next line
+  - contains_over_first_not_nil # Prefer contains over first(where:) != nil
+  # - convenience_type # Types used for hosting only static members should be implemented as a caseless enum to avoid instantiation.
+  - explicit_init # Explicitly calling .init() should be avoided.
+  - explicit_self # Instance variables and functions should be explicitly accessed with 'self.'.
```

Comment:
should we remove this? it is not running as a rule here.  we can like below if we want to use it in the future maybe
```
analyzer_rules: # Rules run by `swiftlint analyze` (experimental)
  - explicit_self
```